sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveDataFrameWindowSuite.scala

```diff
@@ -189,7 +189,7 @@ class HiveDataFrameWindowSuite extends QueryTest {
       df.select(
         $"key",
         last("value").over(
-          Window.partitionBy($"value").orderBy($"key").rangeBetween(1, Long.MaxValue))
+          Window.partitionBy($"value").orderBy($"key").rangeBetween(-1, 0))
```

Comment:
So I have been looking at this, and I am not to sure what is being tested here. It is currently trivial because we are trying to get the LAST_VALUE of a column that is used in the partitioning statement - the result is a constant within a partition. 

@chenghao-intel I think you wrote this test, can you perhaps tell me what is supposed to be tested here?